src/main/java/seedu/address/logic/wanderlustlogic/wanderlustcommands/edit/EditAccommodationCommand.java

```diff
@@ -30,15 +31,16 @@
             + ": Edits the accommodation identified by the index number used in the displayed accommodation list.\n"
             + "Parameters: INDEX (must be a positive integer)\n"
             + "[" + PREFIX_NAME + "NAME] "
+            + "[" + PREFIX_LOCATION + "LOCATION] "
```

Comment:
Do you think we want to make the arrangements of all these fields consistent throughout our whole repo. Cos I feel its a little bit messy right now, everybody's arrangement is different. 
@jiaweiteo @jeannetoh99 @underthehai @timjkong

Reply:
I tried to follow the order given in the UI